rpmalloc-sys/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "rpmalloc-sys"
-version = "0.2.2+1.4.1"
+version = "0.2.2+1.4.3"
```

Oh wait looked closer and the submodule you updated here seems to not point to the `1.4.3` release of rpmalloc, but to the latest commit https://github.com/mjansson/rpmalloc/commit/b097fd0916500439721a114bb9cd8d14bd998683, and there hasn't been a newer official numbered release yet that includes the Mac fixes from https://github.com/mjansson/rpmalloc/pull/275

We can update and release with an in-development `rpmalloc` submodule, but then the `rpmalloc-sys` version needs to specify the commit instead in the metadata like this:

```suggestion
version = "0.2.2+b097fd0"
```

---

Yes, I actually took the latest version. But I agree with you, taking ```b097fd0``` as the version is more correct!